vendor/engines/split_accounts/app/models/split.rb

```diff
@@ -0,0 +1,19 @@
+class Split < ActiveRecord::Base
+
+  belongs_to :parent_split_account, class_name: "SplitAccount", foreign_key: :parent_split_account_id, inverse_of: :splits
+  belongs_to :subaccount, class_name: "Account", foreign_key: :subaccount_id, inverse_of: :parent_splits
+
+  scope :with_extra_penny, -> { where(extra_penny: true) }
```

Comment:
We've been mostly standardizing on the `def self.with_extra_penny` version of scopes.